app/controllers/api/v1/pages_controller.rb

```diff
@@ -22,7 +23,10 @@ def show
   end
 
   def update
-    if page.update(page_params)
+    page.update!(page_params)
+
+    if page.valid?
```

Comment:
could we maybe using `.save` instead of `.update` like we do for create action? 

`.save` creates the record if it doesn't exist or it updates it.

Reply:
I guess so - do we lose anything by doing that though? If an API user wants to create a page, they can use create or update. If we want to add logic in the future which only happens on create would that make it harder? Would logging be more difficult if update can also create pages?

Reply:
I dont think it would make it harder to log events because the user would still be calling the various create/update controller action